src/app/core/entity-components/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -155,6 +164,25 @@ export class EntitySubrecordComponent<T extends Entity>
     }
   }
 
+  ngAfterViewInit() {
+    this.recordsDataSource.sort = this.sort;
```

Comment:
I had not even realized sort is broken. Thanks for adding and fixing that! :smiley: